### PR TITLE
[Feature] Show full name

### PR DESCRIPTION
**Pull request by KaterinaWire**
## What's new in this PR?

Replace the short user name with the full user name.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/UserProfile/ProfileViewControllerViewModel.swift [hunk 1]
```diff
@@ -241,7 +241,7 @@ extension ProfileViewControllerViewModel: ZMUserObserver {
 
 extension ProfileViewControllerViewModel: BackButtonTitleDelegate {
     func suggestedBackButtonTitle(for controller: ProfileViewController?) -> String? {
-        return bareUser.displayName.uppercasedWithCurrentLocale
+        return (bareUser.name ?? "").uppercasedWithCurrentLocale
```
Comment (billypchan) [inline Wire-iOS/Sources/UserInterface/UserProfile/ProfileViewControllerViewModel.swift]: nil should be returned if bareUser.name is nil.

File: Wire-iOS Tests/InputBar/InputBarTests.swift [hunk 2]
```diff
@@ -105,7 +105,7 @@ final class InputBarTests: ZMSnapshotTestCase {
         userWithLongName.name = "Matt loooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong name"
         let text = "Hello @\(userWithLongName.name!)"
         sut.textView.setText(text, withMentions: [Mention(range: (text as NSString).range(of: "@\(userWithLongName.name!)"), user: userWithLongName)])
-        
+        let test = sut.textView.textContainer.lineBreakMode
```
Comment (typfel) [inline Wire-iOS Tests/InputBar/InputBarTests.swift]: debug code?

File: Wire-iOS/Sources/UserInterface/UserProfile/ProfileViewControllerViewModel.swift (hunk 1 above)
Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/UserProfile/ProfileViewControllerViewModel.swift]: Fixed

File: Wire-iOS Tests/InputBar/InputBarTests.swift (hunk 2 above)
Comment (KaterinaWire) [reply Wire-iOS Tests/InputBar/InputBarTests.swift]: Yes...
Removed

File: Wire-iOS/Sources/UserInterface/UserProfile/ProfileViewControllerViewModel.swift [hunk 3]
```diff
@@ -241,7 +241,10 @@ extension ProfileViewControllerViewModel: ZMUserObserver {
 
 extension ProfileViewControllerViewModel: BackButtonTitleDelegate {
     func suggestedBackButtonTitle(for controller: ProfileViewController?) -> String? {
-        return (bareUser.name ?? "").uppercasedWithCurrentLocale
+        guard let fullName = bareUser.name else {
+            return nil
+        }
+        return fullName.uppercasedWithCurrentLocale
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/UserProfile/ProfileViewControllerViewModel.swift]: You simplify this to:
`return bareUser.name?. fullName.uppercasedWithCurrentLocale `

Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/UserProfile/ProfileViewControllerViewModel.swift]: Done

**Pull request merged**